### PR TITLE
conference and event model

### PR DESCRIPTION
**Pull request by bketelsen**
## Description
add baseline conference and event models
## Related Issue

#3

### CODE REVIEW

File: def/conference.go [hunk 1]
```diff
@@ -1,7 +1,20 @@
...
+}
+
+// Event is an instance like GopherCon 2020
+type Event struct {
+	ID        uint
+	Name      string
+	Slug      string
+	StartDate time.Time
+	EndDate   time.Time
+	Location  string
```
Comment (perrito666) [inline def/conference.go:19]: Perhaps we want this to be a type? I imagine this always being a Venue, and a geographical ref like city/state.

File: def/conference.go [hunk 2]
```diff
@@ -1,7 +1,20 @@
...
+// Conference is a brand like GopherCon
 type Conference struct {
-	Name string
+	ID     uint
+	Name   string
+	Events []Event
+}
+
+// Event is an instance like GopherCon 2020
+type Event struct {
```
Comment (perrito666) [inline def/conference.go:13]: A tagline sounds like something we could use? If memory serves gophercons tend to have one, such as `#GoVirCon ~ see you online!`

**Review by perrito666 — APPROVED** [review]
Looks like a nice base, cheers

File: def/conference.go (hunk 1 above)
Comment (bketelsen) [reply def/conference.go:19]: thought this through and came to the conclusion that these structs should just represent basic metadata. Public facing descriptions of this metadata should be editable by non-technical users.  Added #pace14 to capture this.

**Pull request merged**